extensions/sql-bindings/README.md

```diff
@@ -8,6 +8,22 @@ Microsoft SQL Bindings for VS Code enables users to develop Azure Functions with
 
 This extension is bundled into the `SQL Server (MSSQL)` extension for VS Code and will be installed automatically when that extension is updated or installed.
 
+## Getting Started with SQL Bindings
+
+### From object explorer
+* To create an Azure Function from a specific table in object explorer, right-click on a table in a connected server in object explorer and select `Create Azure Function with SQL Binding.` If you have not yet created the Azure Function project, the process will create one for you before populating the Azure Function.
+![Add SQL Binding in command palette](media/CreateAzFuncTableOE.png)
+
+### In an existing Azure Function
+* While the Azure Function C# code definition is open, search the command palette for `Add SQL Binding` to add a SQL binding to the C# Azure Function.
```

Comment:
Would suggest this slight rewording instead - "code definition" isn't really clear and you should include the command prefix as well. 

```
Open the C# Azure Function in an editor and then run the `MS SQL: Add SQL Binding` command from the command palette to add a SQL binding to an existing function.
```